CMake/fletch-tarballs.cmake

```diff
@@ -308,8 +308,19 @@ if (fletch_ENABLE_OpenCV OR fletch_ENABLE_ALL_PACKAGES OR AUTO_ENABLE_CAFFE_DEPE
   set_property(CACHE OpenCV_SELECT_VERSION PROPERTY STRINGS "3.4.0" "4.2.0" "4.5.1")
 
   set(OpenCV_version ${OpenCV_SELECT_VERSION})
-  set(OpenCV_url "http://github.com/Itseez/opencv/archive/${OpenCV_version}.zip")
-  set(OpenCV_dlname "opencv-${OpenCV_version}.zip")
+  # Pulling from pypi for version 4.5.1 since it is SWAP approved
+  if (OpenCV_SELECT_VERSION VERSION_EQUAL 4.5.1)
+    if (fletch_USE_PYPI_OPENCV)
+      set(OpenCV_url "https://files.pythonhosted.org/packages/bb/08/9dbc183a3ac6baa95fabf749ddb531bd26256edfff5b6c2195eca26258e9/opencv-python-${OpenCV_version}.48.tar.gz")
```

Comment:
Since this is a special case that only works for 4.5.1, it might be better to just hard code that version in the path on these two lines, especially since that `.48` would not necessarily apply to other versions if this was generalized in the future.